control_panel_api/authentication.py

```diff
@@ -1,37 +1,70 @@
+import logging
+
+import requests
```

Comment:
Import order

Reply:
Looks ok?  Auto ordered/grouped by system, site, app, in alpha.

Reply:
`requests` surely comes after `django`?

Reply:
Good point, didn't notice.  It was some weird from/import grouping option..